crates/forge/tests/cli/test_cmd/invariant/storage.rs

```diff
@@ -1,6 +1,13 @@
-pragma solidity >0.8.13;
+use super::*;
 
-import "ds-test/test.sol";
+forgetest_init!(
+    #[ignore = "slow"]
```

Comment:
we should re-enable this one with AST seeded fuzz dictionary PR, should be much faster cc @0xrusowsky